composer.json

```diff
@@ -45,7 +45,8 @@
         "phpunit/phpunit": "^5.6",
         "se/selenium-server-standalone": "^2.52",
         "sensio/generator-bundle": "^3.1",
-        "stripe/stripe-php": "^4.1"
+        "stripe/stripe-php": "^4.1",
+        "dama/doctrine-test-bundle": "^1.0"
```

Comment:
yeah just used `composer require --dev dama/doctrine-test-bundle` :yum: 

I can restore the order again :wink: